l2-contracts/contracts/bridge/L2StandardDeployer.sol

```diff
@@ -108,8 +108,10 @@ contract L2StandardDeployer is IL2StandardDeployer, Ownable2StepUpgradeable {
 
         /// backwards compatible event
         emit WithdrawalInitiated(_prevMsgSender, _l1Receiver, l2Token, _amount);
+        /// New Format
         // solhint-disable-next-line func-named-parameters
-        _bridgeBurnData = abi.encodePacked(_chainId, _mintValue, _assetInfo, _prevMsgSender, _data);
+        emit BridgeBurn(_chainId, l2Token, _prevMsgSender, _l1Receiver, _mintValue, _amount);
+        _bridgeMintData = _data;
```

Comment:
Here you are returning the same data that was passed in, that is not good, no?

Reply:
That's the only data required for base use-case, we can encode more params for custom tokens for example, but if we're bridging with current implementation, extra data will be unused.